recipes/hkl/build.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+set -e
+
+test -d m4 || mkdir m4
+# gtkdocize || exit 1
+
+export python=$PYTHON
+export PATH=$PREFIX/bin:$PATH
```

Comment:
Why do you need this?

Reply:
There was something bad going on with the build. I can remove it and try again.

Reply:
Removed now. Resolving.